Corona-Warn-App/src/main/res/layout/tracing_details_access_survey_card.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/tracing_details_survey_card_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:focusable="true"
+        tools:showIn="@layout/home_card_container_layout">
+
+        <TextView
+            android:id="@+id/tracing_details_survey_card_title"
```

Comment:
The title is cut off on my phone and according to the design, the illustration should be displayed until the edge. 

![Screenshot 2021-02-03 at 09 35 06](https://user-images.githubusercontent.com/10398034/106719895-1b118480-6603-11eb-92a7-149750446810.png)

Reply:
good catch, thx
adjusted to:
![image](https://user-images.githubusercontent.com/46747780/106755382-5bd2c300-662e-11eb-8d2d-235c040c05e0.png)